core/git_mixins/active_branch.py

```diff
@@ -86,6 +87,8 @@ def get_branch_status(self, delim=None):
                 secondary = "You're ahead by {}.".format(ahead)
             elif behind:
                 secondary = "You're behind by {}.".format(behind)
+            elif gone:
+                secondary = "The remote branch is gone."
```

Comment:
@divmain Does this message look ok to you? For consistancy with ahead/behind, I could highlight "gone" via `branches.tmLanguage`, maybe something like this:
```xml
<dict>
	<key>comment</key>
	<string>ahead, behind or gone</string>
	<key>match</key>
	<string>(ahead)|(behind)|(gone)</string>
	<key>name</key>
	<string>keyword.other.git-savvy.status.ahead-behind</string>
</dict>
```